pandas/core/arrays/timedeltas.py

```diff
@@ -219,6 +267,19 @@ def _evaluate_with_timedelta_like(self, other, op):
 
         return NotImplemented
 
+    # ----------------------------------------------------------------
+    # Comparison Methods
+
+    @classmethod
+    def _add_comparison_methods(cls):
```

Comment:
so I would define the comparison methods on the class itself, then this should be trivial to change to use the EAMixins

Reply:
I agree we should move in this direction, but prefer to do it separately.  ATM the PR still broadly resembles cut/paste.